src/decision/include/LidarDecision.h

```diff
@@ -48,13 +48,27 @@ class LidarObstacle {
     LidarObstacle(std::vector<Reading> readings);
 
     /**
+    * Gets the distance of the first data from a Obstacle group
+    *
+    * @return the distance of the first data from a Obstacle group
+    */
+    distance_t getFirstDistance();
```

Comment:
Ah, ok, in that case, the `@return` value might be better phrased as: `the distance to the rightmost point of the obstacle`